tests/torch/test_hybrid_converter.py

```diff
@@ -106,40 +123,52 @@ def run_hybrid_llm_test(
 
     # Get the topk indices for logits_disable and logits_simulate
     topk_disable = logits_disable.topk(k, dim=-1).indices
-    topk_simulate = logits_simulate.topk(k, dim=-1).indices
     topk_original = logits_original.topk(k, dim=-1).indices
 
     # Compute accuracy of disable and simulate by checking
     # how many labels correspond with the topk_original
     accuracy_disable = (topk_disable == topk_original).float().mean().item()
-    accuracy_simulate = (topk_simulate == topk_original).float().mean().item()
-
+    # Ensure logits_disable and logits_original return the same output for the logits
     # Assert that both accuracy values are above the expected threshold
     assert (
         accuracy_disable >= expected_accuracy
     ), f"Disable accuracy {accuracy_disable:.4f} is below the expected {expected_accuracy:.4f}"
-    assert (
-        accuracy_simulate >= expected_accuracy
-    ), f"Simulate accuracy {accuracy_simulate:.4f} is below the expected {expected_accuracy:.4f}"
+
+    if logits_simulate is not None:
+        assert torch.allclose(logits_disable, logits_simulate, atol=1e-7), "Outputs do not match!"
+        topk_simulate = logits_simulate.topk(k, dim=-1).indices
+        accuracy_simulate = (topk_simulate == topk_original).float().mean().item()
+        assert accuracy_simulate >= expected_accuracy, (
+            f"Simulate accuracy {accuracy_simulate:.4f} is below "
+            f"the expected {expected_accuracy:.4f}"
+        )
 
     with tempfile.TemporaryDirectory() as temp_dir:
         temp_dir_path = Path(temp_dir)
 
         # Get the temp directory path
-        hybrid_model.save_and_clear_private_info(temp_dir_path)
-        hybrid_model.set_fhe_mode("remote")
 
-        # At this point, the hybrid model does not have
-        # the parameters necessaryto run the module_names
-        module_names = module_names if isinstance(module_names, list) else [module_names]
+        if not has_pbs and glwe_backend_installed:
```

Comment:
don't we have a better way to check if glwe is used?

Reply:
this is not checking, it's enabling/disabling the glwe backend based on the value of `glwe_backend_installed`